app/src/main/java/org/oppia/app/viewmodel/ViewModelBridgeFactory.kt

```diff
@@ -2,16 +2,17 @@ package org.oppia.app.viewmodel
 
 import androidx.lifecycle.ViewModel
 import androidx.lifecycle.ViewModelProvider
+import java.security.Provider
 import javax.inject.Inject
-import javax.inject.Provider
+
 
```

Comment:
Are there are 2 blank lines? If so, remove one of them.

Reply:
resolved